components/brave_ads/browser/component_updater/component_util.cc

```diff
@@ -14,4161 +14,5069 @@ namespace brave_ads {
-      "/4Q0OqiaE1NmwAupM5tFoF6COv3tfpnhuaokxkMb+"
-      "OcvJg5vl3j8DcNlkjvRnHBjpvIqhTnknt4IBWpoGNX7CdDKjCrSurZlGhc3W78KiQl803MHM"
-      "wgaRGSSjaIcBFML7zuwyl3BT6yGq3yFNVXzLKnCQuddbqM2Cfo+"
-      "y8sfRJnh9bkCDXQ00KqU2iLlrPE3RYGWcSTSGQIDAQAB"}},
-    {"yi",
-     {"jhnklldjooclfmgpkipaemehnngabckf",
-      "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwAzCfqsFvkjjce2B34AVC/"
-      "5IE0T7WtkKJnZshG6/3jxHCm0vaOaCvQDFAYFfLOczj+iwr4tR8e2ggYvsY/TvQ/E/zto9/"
-      "CCUdv3v9p7U3AJtUVhzjh48s6Yk7MC+0a5WhaJ+HTXegIK7G1g2Zd/"
-      "wHEHi9UDruqskv9QtX7EgPkAKVzr3po8E5U2NoUdzeRi1DZgGme0iZO58XicB0ZMiXj55pQg"
-      "dPmapkLTabYVwBVBpJDsNr5ne9LWDXxzaxkF1mcMb/"
-      "muGtItD09LHOJuLR6yvmnxX+"
-      "MSaRmK4IvGvBOSrAFy5uzZ16mSHp0AqXIiDeD7tNzL8XO0Ap8ffT8BitQIDAQAB"}},
-    {"yo",
-     {"fjfbodkpnkomodlcanacakhcfmjjgkdf",
-      "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAswzOWUbH820bu5STKuys/d/"
-      "rp86M3cLr9bd9jXamajUIwRhC84n6WrnP+ilKoaHNmlKukL26kZHqahI+"
-      "ApFrPzwwmuk9hk5CzfV2emtj69CXQD1Da67ThHpU2rAOsRPtfKMkgYgEcfufJZyU7OUny9Qx"
-      "zorshEN34d2BOEC8rVEnTzASTeOFDuhIGsXzBVxauG+"
-      "NYneC67Ilc9JbNPc3pcTXceoajtmzA7YAQRQ7QKJg1bnWjFNlefAVjMg7Kuh2PLVSWFCSF+"
-      "Ni/VtUwx/8sf+kS4wNF3tM2oqDoJaOxEPt/2glG1Djzj50pI/oKBeBc0r6wZfe/"
-      "YAWVOWi8tVL2wIDAQAB"}},
-    {"za",
-     {"bncbapkadghlbephbogcmomlecfmdhnb",
-      "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAyOwy2XIcaxQQFZphrTWxTBjZJWRX"
-      "yMO1eirtUEuPLw1pHbjAtv2Psie1JnSUmjBiLs/u/n2ghhPvo6/"
-      "mVhSnTjPSa3ab7SqWD6aVRgnGawskDDLRNcA/"
-      "SFLunbRq3PRep0R9K0kcYxd131KQHEC5GzW+DUQv1jUkqde7y2+hZtFqqIMIxRYOckiittg+"
-      "ur3d8pzP88lEBcYsj109E3LyHyJ31Wbbd8utdVuhnQT1ZkLvOQp94rZHFztckpqy3NeLEKpS"
-      "K4+V6JxGpO5BHo9kIncaAV3QzcZXZz50oIGdw25f10fqmlm+St/AKPCqhQIIp9MpRITJW6G/"
-      "LACD6rsCkwIDAQAB"}},
-    {"zh",
-     {"clegognodnfcpbmhpfgbpckebppbaebp",
-      "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA5gsdeMl6rI/"
-      "OmDF5uxPEMJ+7CyMDDHoi61AZS+cp+DquTRWRvgPgwde5G/"
-      "xihE3kIfcyG9yfRtxH11c7ftTCmbdE+h8Q+"
-      "10ERbP5yHf2ZeYhSr8XqzZdfoppxRpqIerRl4gtPxuEek/"
-      "yLJRBnjaBqbljHOTEe2Wh1tbS3jAWd+joPE1HVU2rz/axo1t/"
-      "M8Ge+IWp38xOu7lss7118imi+RiePg1+dNt+F0aP2IjIw4COAF4CIaMqvhJEh/"
-      "oupZJ6vNsciqJ/xRE+N6bSoZ6xSV4XvCj6S1LKwKa/"
-      "ssyCsWkQtKCClfQKBh3enpB1p4Inwl1sybsLqp/P7Y2ixbHIAQIDAQAB"}},
-    {"zu",
-     {"dhlnknppkgfgehmmipicnlplhjgpnmnh",
-      "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwcXcnw+ttOPKYhEOOMXZv796/"
-      "DUkPBrJgFcI5XwlDtUyasf2dVjE6bALgMIPjYGuVZf/"
-      "FqDxd7FH6EcVR0Yn0wmEJLHNkifn9DrKcjrVBbef8+"
-      "kp7MexC7GQt6U3Y0lBAk4CpLvYAW7nRZnJG4Ngalbf4+ZiM71sWqBFgIOaZWLGTEWTDA7Q/"
-      "PpWHavq/07i+PjYPIClTY5RHnsyt7wVcT9pSt+wiGPTfa/"
-      "vK8w0Bg1b64Gs2YqH51WrJeNbWz+WFaF3iZkpp0yzY9ZvvEt8kcAdf+Gy33DN/"
-      "q+M+iVcyCnFVgtyYQFfOgc6y8Jktz6HzEUrTykk2qLe2NIs9wkSNQIDAQAB"}},
-});
+constexpr auto kComponents = base::MakeFixedFlatMap<std::string_view,
```

Comment:
@cdesouza-chromium @AlexeyBarabash was this change intentional, I ask because this was a recent change. I will hold off reviewing just incase this was a rebase issue and that other changes may be reverted. Thanks

Reply:
@tmancey , sorry, this PR is against the cr121 branch.
`cr121` was force-pushed, so the PR contains a lot of unrelated commits. I Will rebase it now.

Your question would be better to address when `cr121` will be completed and opened for review.

Reply:
Ah :-) Thanks